Gruntfile.js

```diff
@@ -38,7 +38,7 @@ module.exports = (grunt) => {
       resolve: {
         extensions: ['.ts', '.js', '.json']
       },
-      devtool: process.env.VEX_GENMAP || mode === 'production' ? 'source-map' : false,
+      devtool: process.env.VEX_GENMAP || mode === 'production' ? 'source-map' : 'source-map',
```

Comment:
can remove the mode step ? : tests since they evaluate to the same.

Agreed that generating source-maps is just as crucial in development as in implementation.  Takes a little longer to generate but so worth it.

Reply:
I left it in order to decide if a different approach was required between `production` and `development`
if this is not the case I would simply suggest
`devtool: 'source-map'`

Reply:
Thinking about this more, this is really going to slow down the build during the dev workflow. We already have `env.VEX_GENMAP` for folks to use, so maybe this change is unnecessary. You can `VEX_GENMAP=true grunt build` on the commandline.

Reply:
Gruntfile.js reverted